app/services/hyrax/workflow/workflow_importer.rb

```diff
@@ -1,27 +1,41 @@
 module Hyrax
   module Workflow
     class WorkflowImporter
-      # @api public
-      #
-      # Load all the workflows in config/workflows/*.json
-      # @return [TrueClass]
-      def self.load_workflows
-        Dir.glob(Rails.root + "config/workflows/*.json") do |config|
-          Rails.logger.info "Loading workflow: #{config}"
-          generate_from_json_file(path: config)
+      class << self
+        attr_accessor :load_errors
+
+        # @api public
+        #
+        # Load all the workflows in config/workflows/*.json
+        # @return [Array] List of errors during processing
```

Comment:
An intention of the `@api public` is to say that the method signature and return values do not change (without ample deprecation warning).  The new documentation says it returns an Array of load_errors (though the method still returns true).

I believe the return value should remain true.